site/en/education/recommendations-for-web-views.md

```diff
@@ -0,0 +1,48 @@
+
+### Ensure your WebView only serves content you want and does not link out
+
+If your app requires that a webview is within your application, make sure that the content within your webview is not able to escape your application and navigate to content that you do not own or control. This can be done by modifying the [webview client to override loading certain URLs via an allowlist](<https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20android.webkit.WebResourceRequest)>) on Android or by manually parsing through your site and validating that the links do not link out to third party content that you cannot control as this may enable your users to go to inappropriate content. For Chrome Apps, you can use the [WebRequestEventInterface](https://developer.chrome.com/docs/extensions/reference/webviewTag/#type-WebRequestEventInterface) to also modify requests and validate that they are allowed to navigate to your allowed list of sites.
+
+## Android Apps
+
+### Use Chrome Custom Tabs instead of WebViews
+
+By employing Chrome Custom Tabs, you are simply redirecting your users to the Chrome Browser on the device that they are using. This is most useful for when you may have a third party identity provider that relies on implementing OAuth in a webview. For Chrome OS, this will launch the normal browser and allow users to navigate within the page as normal but it will allow the enforcement of browser policies to the content that they students are using. For most educational organizations, this means that the content that would be displayed would be visible to extensions running on the browser including content filtering.
+
+### Using Google Sign In without an additional identity provider
+
+If your Android application only uses Google Sign in, do not redirect to the webview to sign in but rather handle it with the [One Tap sign-in and sign-up flow](https://developers.google.com/identity/one-tap/android). Links in this sign-in flow start a browser intent rather than an embedded webview so your data can be viewed in the Chrome browser of the device, not a webview which may not have the same filtering controls applied to it.
+
+## Chrome Apps
+
+### Launch External Sites in the browser rather than an embedded webview
+
+If you have content such as help articles or other content that is not owned or controlled by you, it would help to link to that article in a new tab rather than open within the webview that is running. To open a new tab from your Chrome App, you can modify your anchor tags to set the [target attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) to point to `\_blank` which will allow the link to open in a new tab.
```

Comment:
```suggestion
If you have content such as help articles or other content that is not owned or controlled by you, it would help to link to that article in a new tab rather than open within the WebView that is running. To open a new tab from your Chrome App, you can modify your anchor tags to set the [target attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) to point to `_blank` which will allow the link to open in a new tab.
```